packages/client/src/views/SearchResult/SearchResult.tsx

```diff
@@ -412,7 +412,9 @@ export class SearchResultView extends React.Component<
               <Query
                 query={SEARCH_EVENTS}
                 variables={{
-                  locationIds: userDetails
+                  locationIds: this.userHasRegisterScope()
+                    ? null
+                    : userDetails
```

Comment:
I am not sure why the coverage fails but looks like you can add a unit test for this scenario.